### PR TITLE
BZ-1125077 - upgrading code to default behavior in 7.x

### PR DESCRIPTION
**Pull request by mrietveld**
@mbiarnes Could you review this when you get a chance?

### CODE REVIEW

File: jbpm-console-ng-workbench-integration/jbpm-console-ng-workbench-integration-client/src/test/java/org/jbpm/console/ng/wi/client/editors/deployment/descriptor/DeploymentDescriptorViewTest.java [hunk 1]
```diff
@@ -201,8 +201,10 @@ public void changeDefaultLimitSerializationClassesValueToTrueIn7x() throws Excep
 
         viewImpl.setContent(deploymentDescriptorModel);
 
+        boolean limitSerializationClasses = DeploymentDescriptorViewImpl.getLimitSerializationClassesCheckBoxValue(deploymentDescriptorModel);
+
         assertTrue( "The default value of 'limitSerializationClasses is FALSE in 6.x and TRUE in 7.x",
-                jarVersion < 7.0d && ! viewImpl.limitSerializationClassesCheckBox.getValue()
-                || jarVersion >= 7.0d && viewImpl.limitSerializationClassesCheckBox.getValue() );
+                ( jarVersion < 7.0d && ! limitSerializationClasses )
+                || (jarVersion >= 7.0d && limitSerializationClasses ) );
```
Comment (mswiderski) [inline jbpm-console-ng-workbench-integration/jbpm-console-ng-workbench-integration-client/src/test/java/org/jbpm/console/ng/wi/client/editors/deployment/descriptor/DeploymentDescriptorViewTest.java]: can we have similar fix for jbpm that has that test too?

Comment (mrietveld) [inline jbpm-console-ng-workbench-integration/jbpm-console-ng-workbench-integration-client/src/test/java/org/jbpm/console/ng/wi/client/editors/deployment/descriptor/DeploymentDescriptorViewTest.java]: Yup! :D 
https://github.com/droolsjbpm/jbpm/pull/382

**Comment by mswiderski** [comment]
was this really meant for @mbiarnes?

anyway, +1 from me as the build is green here

**Comment by mrietveld** [comment]
@mswiderski Please merge when you get a chance. Thanks!

**Pull request closed without merging**